examples/aws/terraform/ha-autoscale-cluster/vars.tf

```diff
@@ -38,6 +38,12 @@ variable "node_instance_type" {
   default = "t4g.medium"
 }
 
+// Instance type used for bastion server
+variable "bastion_instance_type" {
+  type    = string
+  default = "t4g.medium"
```

Comment:
In the interest of having a more "friendly" experience for new users, maybe we could default this to null, then conditionally set the instance type deployed based off of the AMI architecture from `data.aws_ami.base`?

Reply:
I really like this idea, but think it's probably better implemented in a future PR as it'll require a fair bit more testing.